### PR TITLE
Origin/static/component tables

### PR DESCRIPTION
**Pull request by mutaharaslam-ml**

### CODE REVIEW

File: apps/angular-demos/src/app/app.module.ts [hunk 1]
```diff
@@ -40,6 +40,7 @@ import { NoIconComponent } from './examples/card/no-icon/no-icon.component';
 import { WithImageComponent } from './examples/card/with-image/with-image.component';
 import { ImageWithSolidShadowComponent } from './examples/card/image-with-solid-shadow/image-with-solid-shadow.component';
 import { AdvanceStepperComponent } from './examples/stepper/advance-stepper/advance-stepper.component';
+import { CommonModule } from '@angular/common';
```
Comment (zeenainali) [inline apps/angular-demos/src/app/app.module.ts:43]: Remove this as well.

File: package.json [hunk 2]
```diff
@@ -102,6 +102,7 @@
     "jsdom": "^16.2.2",
     "lioness": "^0.2.17",
     "mini-css-extract-plugin": "^0.9.0",
+    "ngx-toastr": "^13.2.0",
```
Comment (zeenainali) [inline package.json:105]: this one too

File: libs/static/src/scss/components/_tables.scss [hunk 3]
```diff
@@ -0,0 +1,141 @@
+:root {
+  --striped-light: #e5e5e5;
+  --striped-dark: #333333;
+}
+
+.table {
+  border-collapse: collapse;
+  padding: 0;
+  margin: 0 0 11px 0;
```
Comment (zeenainali) [inline libs/static/src/scss/components/_tables.scss:9]: Use margin VAR or use specing VAR

File: libs/static/src/scss/components/_tables.scss [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+  width: 100%;
+
+  th,
+  td {
+    border-bottom: 1px solid var(--light);
+    text-align: left;
+    padding: 10px;
+    vertical-align: bottom;
```
Comment (zeenainali) [inline libs/static/src/scss/components/_tables.scss:17]: Vertical align should be available with class, so they can change based on the need.

File: libs/static/src/scss/components/_tables.scss [hunk 5]
```diff
@@ -0,0 +1,141 @@
...
+    color: var(--dark);
+
+    &td {
+      vertical-align: top;
```
Comment (zeenainali) [inline libs/static/src/scss/components/_tables.scss:21]: This one should be based on the class.

File: libs/static/src/scss/components/_tables.scss [hunk 6]
```diff
@@ -0,0 +1,141 @@
...
+    }
+  }
+
+  thead {
+    th {
+      font-weight: bold;
```
Comment (zeenainali) [inline libs/static/src/scss/components/_tables.scss:27]: Use VAR for bold from typography.

File: libs/static/src/scss/components/_tables.scss [hunk 7]
```diff
@@ -0,0 +1,141 @@
...
+    }
+  }
+
+  &--dark {
+    background-color: var(--dark);
+
+    th,
+    td {
+      color: var(--light);
+      border-bottom: 1px solid var(--light);
```
Comment (zeenainali) [inline libs/static/src/scss/components/_tables.scss:53]: Based on the theme only change the border color, border width should be generic.

File: libs/static/src/scss/components/_tables.scss [hunk 8]
```diff
@@ -0,0 +1,141 @@
...
+            background-color: var(--striped-dark);
+          }
+        }
+      }
+    }
+  }
+}
+
+/* responsive .table */
+@media screen and (max-width: 992px) {
```
Comment (zeenainali) [inline libs/static/src/scss/components/_tables.scss:108]: Use default media query.

**Review by zeenainali — CHANGES_REQUESTED** [review]

File: libs/static/src/scss/components/_tables.scss (hunk 4 above)
Comment (mutaharaslam-ml) [reply libs/static/src/scss/components/_tables.scss:17]: it's generic for all the td's, otherwise, i would need to give it on all td's used in all tables

Comment (zeenainali) [reply libs/static/src/scss/components/_tables.scss:17]: But we are not sure what user need in future so we have to cover all the use cases.

**Pull request closed without merging**